### ISSUE TITLE
Register the package

### ISSUE THREAD

**Issue by ufechner7** [Clemapfel/mousetrap.jl#13]
When at least the main examples work on at least one OS, please register the package to make it easier to install. You can also create a meta-package that installs all dependencies.

**Comment by Clemapfel** [comment]
Yes, this is true. As it says in the [planned features](https://github.com/Clemapfel/mousetrap.jl#planned-features) making the installation a one-liner is a high priority

**Comment by Clemapfel** [comment]
I'm just waiting for https://github.com/JuliaPackaging/Yggdrasil/pull/7461 now, everything is in order and ready to be published on Mousetraps end. Once the jll is on yggdrasil, I can open the Julia registry PR.

**Comment by ufechner7** [comment]
Now merged.

**Comment by ufechner7** [comment]
Writing `@JuliaRegistrator register()` should do the job, shouldn't it?

**Comment by LilithHafner** [comment]
Bump. This is a great package! It is _way_ more stable and functional than most packages when they are first registered in the General registry. Registering a package also does not come with any additional commitments to no longer releasing breaking changes or anything else. I don't see any reason not to register this package now.
